deepchecks/nlp/utils/text_properties.py

```diff
@@ -875,6 +696,8 @@ def calculate_builtin_properties(
         Also, if a folder already contains relevant resources they are not re-downloaded.
     batch_size : int, default 8
         The batch size.
+    cache_models : bool, default False
+        cache the models being used in this function, to save load time in next execution
```

Comment:
would add more details, what are the memory requirements

Reply:
The memory is vary between model and model...